runelite-client/src/main/java/net/runelite/client/plugins/devtools/ScriptInspector.java

```diff
@@ -376,6 +422,42 @@ private void addScriptLog(ScriptTreeNode treeNode)
 			}
 			DefaultTreeModel treeModel = new DefaultTreeModel(treeNode);
 			JTree tree = new JTree(treeModel);
+			tree.addMouseListener(new MouseAdapter()
+			{
+				@Override
+				public void mouseClicked(MouseEvent e)
+				{
+					if (e.getButton() == MouseEvent.BUTTON3)
+					{
+						int row = tree.getClosestRowForLocation(e.getX(), e.getY());
+						tree.setSelectionRow(row);
+						JPopupMenu popup = new JPopupMenu();
+						JMenuItem item = new JMenuItem("Copy args");
+						item.addActionListener(new ActionListener()
+						{
+							@Override
+							public void actionPerformed(ActionEvent e)
+							{
+								DefaultMutableTreeNode node = (DefaultMutableTreeNode) tree.getLastSelectedPathComponent();
+								// example entry: 69  -  378.62 args: [69, 0, 24772672]
+								Pattern argPattern = Pattern.compile("args: \\[(.*)]");
+								Matcher matcher = argPattern.matcher(node.toString());
+								if (matcher.find())
+								{
+									Clipboard clipboard = Toolkit.getDefaultToolkit().getSystemClipboard();
+									clipboard.setContents(new StringSelection(matcher.group(1)), null);
```

Comment:
Is there any reason we're formatting it as: `2282, 11927571, 11927572, 11927573, 11927574, 11927575` instead of `2282  -  182.18 Logout.SATISFACTION_5SCALE args: [2282, 11927571, 11927572, 11927573, 11927574, 11927575]`? The latter seems better, imo.